sway/server.c

```diff
@@ -30,6 +32,14 @@
 #include "sway/xwayland.h"
 #endif
 
+static int sigchld_handler(int signum, void *data) {
+	while (waitpid(0, NULL, WNOHANG) > 0) {
+		// Do nothing
```

Comment:
similarily I think this should match against known pids (e.g. swaybg_pid etc) and unset the values if it died, so we don't kill a random process later if a user killed swaybg themselves.